src/plugins/kibana/server/lib/schemas/resources/index_pattern_schema.js

```diff
@@ -6,6 +6,7 @@ module.exports = Joi.object({
   time_field_name: Joi.string(),
   interval_name: Joi.string(),
   not_expandable: Joi.boolean(),
+  fieldFilters: Joi.string().default('[]'),
```

Comment:
BTW, where does the translation from the new Kibana API to the format stored in Elasticsearch take place ?